src/pint/models/noise_model.py

```diff
@@ -429,23 +429,23 @@ def __init__(
 
         self.add_param(
             floatParameter(
-                name="TNRedAmp",
+                name="TNREDAMP",
                 units="",
                 aliases=[],
                 description="Amplitude of powerlaw " "red noise in tempo2 format",
             )
         )
         self.add_param(
             floatParameter(
-                name="TNRedGam",
+                name="TNREDGAM",
```

Comment:
Is this an unrelated bug fix? It'd be good to make this a separate PR so it can be merged sooner.